examples/ibm-cluster/vpc-cluster-vol-attachment/README.md

```diff
@@ -0,0 +1,136 @@
+# IBM VPC Cluster Volume Attchment 
+
+This example shows how to attach block storage volume to VPC Cluster's worker node. To attach storage volume blocks to different worker nodes, change the worker node ID.
+ 
+Note : 
+1. When a VPC storage volume is attached to the worker node, It can not be shared/attached to the other worker nodes.
+2. The worker node and the storage volume should be in the same zone.
+
+Following types of resources are supported:
+
+* [ VPC Worker Storage ](https://cloud.ibm.com/docs/terraform?topic=terraform-container-resources#vpc-gen2)
+
+
+## Terraform versions
+
+Terraform 0.12. Pin module version to `~> v1.21.0`. Branch - `master`.
+
+## Usage
+
+To run this example you need to execute:
+
+```bash
+$ terraform init
+$ terraform plan
+$ terraform apply
+```
+
+Run `terraform destroy` when you don't need these resources.
+
+## Example Usage
+
+Create a container cluster:
+
+```hcl
+
+provider "ibm" {
+  generation = 2
+}
+
+resource "ibm_is_vpc" "vpc1" {
+  name = "testvpc"
+}
+
+resource "ibm_is_subnet" "subnet1" {
+  name                     = "test-subnet1"
+  vpc                      = ibm_is_vpc.vpc1.id
+  zone                     = "us-south-1"
+  total_ipv4_address_count = 256
+}
+
+resource "ibm_is_subnet" "subnet2" {
+  name                     = "test-subnet2"
+  vpc                      = ibm_is_vpc.vpc1.id
+  zone                     = "us-south-2"
+  total_ipv4_address_count = 256
+}
+
+data "ibm_resource_group" "resource_group" {
+  name = "Default"
+}
+
+resource "ibm_container_vpc_cluster" "cluster" {
+  name              = "testcluster"
+  vpc_id            = ibm_is_vpc.vpc1.id
+  kube_version      = "1.18"
+  flavor            = "bx2.2x8"
+  worker_count      = 3
+  resource_group_id = data.ibm_resource_group.resource_group.id
+
+  zones {
+    subnet_id = ibm_is_subnet.subnet1.id
+    name      = "us-south-1"
+  }
+}
+
+resource "ibm_is_volume" "storage_block"{
+    name = var.volume_name
+    profile = var.volume_profile
+    zone = "us-south-1"
+}
+
+data "ibm_container_vpc_cluster" "cluster_info"{
+    name = ibm_container_vpc_cluster.cluster.name
+}
+
+resource "ibm_container_vpc_worker_storage" "volume_attach"{
+    volume = ibm_is_volume.storage_block.id
+    cluster = ibm_container_vpc_cluster.cluster.id
+    worker = data.ibm_container_vpc_cluster.cluster_info.workers[0]
+}
+
+```
+
+```hcl
+data "iibm_container_vpc_worker_storage" "cluster" {
```

Comment:
Is this right: `iibm_container_vpc_worker_storage` . Shouldn't it be: `ibm_container_vpc_worker_storage`